[Bug] [Master] Master frequent print logs

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

**1. No worker:**
[INFO] 2022-04-08 06:14:58.688 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[309] - process event: State Event : key : null type: TASK_STATE_CHANGE executeStatus: NEED_FAULT_TOLERANCE task instance id: 8 process instance id: 7 context: null
**2. Worker high load**
fail to execute : ... due to no suitable worker,  current task needs worker group ... to execute
**3. The running task was deleted**
[INFO] 2022-04-06 11:13:46.749 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[301] - process event: State Event : key : null type: TASK_TIMEOUT executeStatus: null task instance id: 10277 process instance id: 6 context: null
[ERROR] 2022-04-06 11:13:46.749 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[264] - state handle error:
java.lang.NullPointerException: null

### What you expected to happen

Master should not print logs frequently.

### How to reproduce

Please take a look

### Anything else

_No response_

### Version

2.0.5

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-ui/src/common/common.ts
+++ b/dolphinscheduler-ui/src/common/common.ts
@@ -31,7 +31,8 @@ import {
   StopOutlined,
   GlobalOutlined,
   IssuesCloseOutlined,
-  SendOutlined
+  SendOutlined,
+  HistoryOutlined
 } from '@vicons/antd'
 import { format, parseISO } from 'date-fns'
 import _ from 'lodash'
@@ -275,6 +276,14 @@ export const tasksState = (t: any): ITaskStateConfig => ({
     icon: SendOutlined,
     isSpin: false,
     classNames: 'dispatch'
+  },
+  PENDING: {
+    id: 18,
+    desc: `${t('project.workflow.pending')}`,
+    color: '#5101be',
+    icon: HistoryOutlined,
+    isSpin: false,
+    classNames: 'pending'
   }
 })
 

--- a/dolphinscheduler-ui/src/common/types.ts
+++ b/dolphinscheduler-ui/src/common/types.ts
@@ -33,6 +33,7 @@ export type ITaskState =
   | 'FORCED_SUCCESS'
   | 'SERIAL_WAIT'
   | 'DISPATCH'
+  | 'PENDING'
 
 export type ITaskStateConfig = {
   [key in ITaskState]: {

--- a/dolphinscheduler-ui/src/locales/modules/en_US.ts
+++ b/dolphinscheduler-ui/src/locales/modules/en_US.ts
@@ -498,6 +498,7 @@ const project = {
     forced_success: 'Forced success',
     serial_wait: 'Serial wait',
     dispatch: 'Dispatch',
+    pending: 'Pending',
     executing: 'Executing',
     startup_type: 'Startup Type',
     complement_range: 'Complement Range',

--- a/dolphinscheduler-ui/src/locales/modules/zh_CN.ts
+++ b/dolphinscheduler-ui/src/locales/modules/zh_CN.ts
@@ -495,6 +495,7 @@ const project = {
     forced_success: '强制成功',
     serial_wait: '串行等待',
     dispatch: '派发',
+    pending: '挂起',
     executing: '正在执行',
     startup_type: '启动类型',
     complement_range: '补数范围',

